[2024-02-28 09:00:15,008] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: defaultargs.tsk3 2024-02-28T08:00:00+00:00 [queued]>
[2024-02-28 09:00:15,074] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: defaultargs.tsk3 2024-02-28T08:00:00+00:00 [queued]>
[2024-02-28 09:00:15,075] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-02-28 09:00:15,076] {taskinstance.py:1096} INFO - Starting attempt 1 of 2
[2024-02-28 09:00:15,076] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-02-28 09:00:15,102] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): tsk3> on 2024-02-28T08:00:00+00:00
[2024-02-28 09:00:15,125] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'defaultargs', 'tsk3', '2024-02-28T08:00:00+00:00', '--job-id', '144', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/defaultargs.py', '--cfg-path', '/tmp/tmpcuogid0z', '--error-file', '/tmp/tmpen46g_k4']
[2024-02-28 09:00:15,115] {standard_task_runner.py:52} INFO - Started process 104 to run task
[2024-02-28 09:00:15,127] {standard_task_runner.py:77} INFO - Job 144: Subtask tsk3
[2024-02-28 09:00:15,245] {logging_mixin.py:109} INFO - Running <TaskInstance: defaultargs.tsk3 2024-02-28T08:00:00+00:00 [running]> on host 14529f4830f2
[2024-02-28 09:00:15,372] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=jezandre_tiago@hotmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=defaultargs
AIRFLOW_CTX_TASK_ID=tsk3
AIRFLOW_CTX_EXECUTION_DATE=2024-02-28T08:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-02-28T08:00:00+00:00
[2024-02-28 09:00:15,377] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2024-02-28 09:00:15,379] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 5']
[2024-02-28 09:00:15,405] {subprocess.py:74} INFO - Output:
[2024-02-28 09:00:20,413] {subprocess.py:82} INFO - Command exited with return code 0
[2024-02-28 09:00:20,493] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=defaultargs, task_id=tsk3, execution_date=20240228T080000, start_date=20240228T090015, end_date=20240228T090020
[2024-02-28 09:00:20,555] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-02-28 09:00:20,610] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
